packages/bento-design-system/src/util/ConfigurableTypes.ts

```diff
@@ -0,0 +1,13 @@
+import { Object } from "ts-toolbelt";
+
+interface ConfigurableTypes {
+  LocalizedString: string;
```

Comment:
we could also include `number` here while at it?
```suggestion
  LocalizedString: string | number;
```

Reply:
done!